content/en/docs/Responses/AlbumID3.md

```diff
@@ -11,19 +11,66 @@ description: >
 {{< tab header="**Example**:" disabled=true />}}
 {{< tab header="OpenSubsonic" lang="json">}}
 {
-  "id": "ad0f112b6dcf83de5e9cae85d07f0d35",
-  "name": "8-bit lagerfeuer",
-  "artist": "pornophonique",
-  "year": 2007,
-  "coverArt": "al-ad0f112b6dcf83de5e9cae85d07f0d35_640a93a8",
-  "starred": "2023-03-22T01:51:06Z",
-  "duration": 1954,
-  "playCount": 97,
-  "created": "2023-03-10T02:19:35.784818075Z",
-  "artistId": "91c3901ac465b9efc439e4be4270c2b6",
-  "songCount": 8,
-  "played": "2023-03-28T00:45:13Z",
-  "userRating": 4
+    "id": "ad0f112b6dcf83de5e9cae85d07f0d35",
+    "name": "8-bit lagerfeuer",
+    "artist": "pornophonique",
+    "year": 2007,
+    "coverArt": "al-ad0f112b6dcf83de5e9cae85d07f0d35_640a93a8",
+    "starred": "2023-03-22T01:51:06Z",
+    "duration": 1954,
+    "playCount": 97,
+    "genre": "Hip-Hop",
+    "created": "2023-03-10T02:19:35.784818075Z",
+    "artistId": "91c3901ac465b9efc439e4be4270c2b6",
+    "songCount": 8,
+    "played": "2023-03-28T00:45:13Z",
+    "userRating": 4,
+    "recordLabels": [
+        {
+            "name": "Sony"
```

Comment:
Can we put id's into labels. They're also tracked with mbid's and the name can have many duplicates so name is too generic

Reply:
The structure is prepared for future evolution, but there's no concept of recordLabel ids in Subsonic at all so it would require a major change to enforce.
If you talk about MB ids then there's no tag for those so require MB usage + server having the proper data.
This is not in the scope of this proposal and a new one can be made after.